internal/jimmhttp/http_test.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2024 Canonical.
+
+package jimmhttp_test
+
+import (
+	"bytes"
+	"context"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	qt "github.com/frankban/quicktest"
+
+	"github.com/canonical/jimm/v3/internal/jimmhttp"
+)
+
+func TestHTTPHandler(t *testing.T) {
+	c := qt.New(t)
+
+	hnd := &jimmhttp.HTTPHandler{
+		HTTPProxier: testHTTPServer{t: c},
+	}
+
+	srv := httptest.NewServer(hnd)
+	c.Cleanup(srv.Close)
+
+	// test ok
+	resp, err := http.Get(srv.URL)
+	c.Assert(err, qt.IsNil)
+	defer resp.Body.Close()
```

Comment:
I think you don't need to call `resp.Body.Close()` if you don't look at the `Body` field. From the `Response.Body` docs:

> The response body is streamed on demand as the Body field is read. If the network connection fails or the server terminates the response, Body.Read calls return an error.

Reply:
the lint complains about it, even if I don't use it